.github/workflows/quality-check.yaml

```diff
@@ -21,11 +21,34 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8]
-          tensorflow: ["~=2.5.0"]
+          python-version: ["3.7", "3.8", "3.9", "3.10"]
+          tensorflow: ["==2.5", "==2.6", "==2.7", "==2.8"]
```

Comment:
Apologies, I'm not very familiar with this syntax. The previous version specifier `~=` allowed to install the latest patch updates. Do we get the same behaviour with this?

Reply:
> Apologies, I'm not very familiar with this syntax. The previous version specifier `~=` allowed to install the latest patch updates. Do we get the same behaviour with this?

Hmm, I thought it did, but apparently not. I'll fix this.